src/NzbDrone.Core.Test/IndexerTests/NewznabTests/NewznabRequestGeneratorFixture.cs

```diff
@@ -204,5 +204,48 @@ public void should_fallback_to_q()
             pageTier2.Url.Query.Should().NotContain("imdbid=0076759");
             pageTier2.Url.Query.Should().Contain("q=");
         }
+
+        [Test]
+        public void should_encode_raw_title()
+        {
+            _capabilities.SupportedMovieSearchParameters = new[] { "q" };
+            _capabilities.TextSearchEngine = "raw";
+
+            MovieSearchCriteria movieRawSearchCriteria = new MovieSearchCriteria
+            {
+                Movie = new Movies.Movie { Title = "Some Movie & Title: Words", Year = 2021, TmdbId = 123 },
+                SceneTitles = new List<string> { "Some Movie & Title: Words" }
+            };
+
+            var results = Subject.GetSearchRequests(movieRawSearchCriteria);
+
+            var page = results.GetTier(0).First().First();
+
+            page.Url.Query.Should().Contain("q=");
+            page.Url.Query.Should().NotContain(" & ");
+            page.Url.Query.Should().Contain("%26");
+        }
+
+        [Test]
+        public void should_use_clean_title_and_encode()
+        {
+            _capabilities.SupportedMovieSearchParameters = new[] { "q" };
+            _capabilities.TextSearchEngine = "sphinx";
+
+            MovieSearchCriteria movieRawSearchCriteria = new MovieSearchCriteria
+            {
+                Movie = new Movies.Movie { Title = "Some Movie & Title: Words", Year = 2021, TmdbId = 123 },
+                SceneTitles = new List<string> { "Some Movie & Title: Words" }
+            };
+
+            var results = Subject.GetSearchRequests(movieRawSearchCriteria);
+
+            var page = results.GetTier(0).First().First();
+
+            page.Url.Query.Should().Contain("q=");
```

Comment:
Same thing here, check that it contains `q=xyz%20123` `or whatever it should actually be.